# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/browser/browser_103_telemetry.js
# Commit: 33f21aec0e20
# Full Hash: 33f21aec0e20d39623554816399ee76d13d7e08f
# Author: edguloien <eguloien@mozilla.com>
# Date: 2022-08-23 21:47:27
# Regressor Bug: 1713203
# File Overlap Count: 1
# Description:
#   Bug 1713203 - Added check for host, scheme and origin attributes check before cookie change broadcast to content processes r=dveditz,dragana,nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146346
# ==============================================================================

diff -r fe25d4f35ef8 -r 33f21aec0e20 netwerk/test/browser/browser_103_telemetry.js
--- a/netwerk/test/browser/browser_103_telemetry.js	Tue Aug 23 17:30:19 2022 +0000
+++ b/netwerk/test/browser/browser_103_telemetry.js	Tue Aug 23 17:36:24 2022 +0000
@@ -101,3 +101,7 @@
 
   gBrowser.removeCurrentTab();
 });
+
+add_task(async function cleanup() {
+  Services.prefs.clearUserPref("dom.securecontext.allowlist");
+});